.env.example

```
@@ -88,3 +88,4 @@ GRAFANA_CLOUD_LOKI_TOKEN=
 LOG_ENV=prod
 # Distinct host label for queries (example: do-fra1-prod-01)
 LOG_HOST=do-fra1-prod-01
+APP_SEO_CRUX_API_KEY=your_crux_api_key_here
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)